src/internal/Rgb48Color.cpp

```diff
@@ -135,9 +135,9 @@ void Rgb48Color::Lighten(uint16_t delta)
 
 Rgb48Color Rgb48Color::LinearBlend(const Rgb48Color& left, const Rgb48Color& right, float progress)
 {
-    return Rgb48Color( left.R + (((int32_t)right.R - left.R) * progress),
-        left.G + (((int32_t)right.G - left.G) * progress),
-        left.B + (((int32_t)right.B - left.B) * progress));
+    return Rgb48Color( left.R + (((float)right.R - (float)left.R) * progress),
```

Comment:
I ran tests on these yesterday, this worked for me on AVR.  Did it not work for you?  
The issue of using float is that floating point math is expensive in CPU cycles, the calculation of subtraction in this case is all integer math, which will get cast to the floating point for the multiplication.